py/client/pydeephaven/_batch_assembler.py

```diff
@@ -17,6 +17,7 @@ def batch(self):
         return self.grpc_table_ops
 
     def build_batch(self):
```

Comment:
typehint

Reply:
Many of these oneline pydocs actually have return values.  It seems like the docs should also have the return values.